README.md

```diff
@@ -1,107 +1,78 @@
-| GCP      | Compute         | compute\_firewall\_no\_ingress\_22          | VPC firewall rules should not permit ingress from '0.0.0.0/0' to port 22 (SSH)                             |
-| GCP      | Compute         | compute\_firewall\_no\_ingress\_3389        | VPC firewall rules should not permit ingress from '0.0.0.0/0' to port 3389 (RDP)                           |
-| GCP      | Compute         | compute\_subnet\_private\_google\_access    | VPC subnet 'Private Google Access' should be enabled                                                       |
-| GCP      | Compute         | compute\_subnet\_flow\_log\_enabled         | VPC subnet flow logging should be enabled                                                                  |
-| Azure    | Storage Account | storage\_account\_deny\_access              | Storage accounts should deny access from all networks by default                                           |
-| Azure    | Storage Account | storage\_account\_microsoft\_services       | Storage accounts 'Trusted Microsoft Services' access should be enabled                                     |
-| Azure    | Storage Account | storage\_account\_secure\_transfer          | Storage accounts 'Secure transfer required' should be enabled                                              |
-| Azure    | Blob Storage    | storage\_container\_private\_access         | Storage containers should have access set to 'private'                                                     |
-| Azure    | Virtual Network | network\_security\_group\_no\_inbound\_22   | Network security group rules should not permit ingress from '0.0.0.0/0' to port 22 (SSH)                   |
-| Azure    | Virtual Network | network\_security\_group\_no\_inbound\_3389 | Network security group rules should not permit ingress from '0.0.0.0/0' to port 3389 (RDP)                 |
-| Azure    | SQL Server      | sql\_server\_firewall\_no\_inbound\_all     | SQL Server firewall rules should not permit ingress from 0.0.0.0/0 to all ports and protocols              |
-
-## Running Regula locally
-
-Install the prerequisites:
+-   Generates a report with the results of all relevant rules and [control mappings](#compliance-controls-vs-rules).
 
+## Getting Started with Regula
+
+### Running Regula locally
+
+Clone the repository and install the prerequisites:
 - [OPA](https://www.openpolicyagent.org/docs/latest/#1-download-opa)
 - [Terraform 0.12+](https://www.terraform.io/downloads.html)
 
-### macOS and Linux
+#### macOS and Linux
 
 Run the following command:
 
-    ./bin/regula [TERRAFORM_PATH] [REGO_PATHS...]
+    ./bin/regula [IAC_PATH] [REGO_PATHS...]
 
-`TERRAFORM_PATH` is the directory where your Terraform configuration files are
+`IAC_PATH` is the directory where your CloudFormation or Terraform configuration files are
```

Comment:
We should specify that `IAC_PATH` should either be a CloudFormation template (YAML or JSON?) or a Terraform directory